integration-tests/gatsby-source-wordpress/__tests__/index.js

```diff
@@ -66,22 +67,24 @@ const isWarmCache = process.env.WARM_CACHE
 const testOnColdCacheOnly = isWarmCache ? test.skip : test
 
 describe(`[gatsby-source-wordpress] Build default options`, () => {
-  beforeAll(done => {
-    ;(async () => {
-      console.log(`Waiting for WPGraphQL to be ready...`)
-      await urling({ url: `http://localhost:8001/graphql`, retry: 100 })
-      console.log(`WPGraphQL is ready`)
-      console.log(`Waiting for plugins to be ready...`)
-      await pluginsAreReady()
-      console.log(`Plugins are ready`)
-
-      if (isWarmCache) {
-        done()
-      } else {
-        gatsbyCleanBeforeAll(done)
-      }
-    })()
-  })
+  beforeAll(
+    withGlobalStore(done => {
```

Comment:
I think the tests and test utils shouldn't need withGlobalStore since they're int tests that are closer to e2e tests. `gatsby develop` is run in a subprocess and then tests make assertions against queries to the gql server so any store scoping will happen inside that subprocess.
I could be missing something of course, did this fix an issue?

Reply:
Oh is it cause the tests weirdly use `fetchGraphQL` from within source-wordpress?

Reply:
Yeah, exactly. I've added it in places where we're doing deep imports from the plugin

Reply:
Maybe it would be better to refactor out those imports 🤔